homeassistant/components/smhi/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""
+Component for the swedish weather institute weather service.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/weather.smhi/
```

Comment:
This is the SMHI component, so weather should not be in the url. Look at other components for the correct link format.